src/modules/enrollment/components/EnrollmentDetails.tsx

```diff
@@ -121,6 +121,28 @@ const EnrollmentDetails = ({
       );
     }
 
+    if (enrollment.sourceCeReferral) {
+      const referral = enrollment.sourceCeReferral;
+      const referredOn = parseAndFormatDate(referral.createdAt);
+
+      content['Referral Source'] = (
+        <Stack gap={1}>
+          {`Referred from ${referral.sourceProjectName} on ${referredOn}`}
```

Comment:
Re: note on the backend PR, needing to accommodate unknown project due to db schema. Right now it technically can come up I think if enrollments are deleted, so better to display something than have the page crash.

```suggestion
          {`Referred from ${referral.sourceProjectName || 'Unknown Project'} on ${referredOn}`}
```